### ISSUE TITLE
Consider using a CHANGELOG

### ISSUE THREAD

**Issue by ErichDonGubler** [rjs-team/rjs#9]
This is a good example of what we might do once there's an actual release and/or interest:

http://keepachangelog.com/en/1.0.0/

**Comment by ricochet1k** [comment]
Yeah, I'm definitely not going to bother with this until there's a release.
